targets/standalone/index.js

```diff
@@ -3,30 +3,18 @@
 /**
  * Internal dependencies
  */
-import { initHappychat, eventAPI } from 'src';
+import api from 'src/api';
 import config from 'targets/standalone/config';
 
 const wpcomOAuth = require( 'wpcom-oauth-cors' )( config( 'oauth_client_id' ) );
 
-const accessToken = () =>
-	new Promise( ( resolve, reject ) => {
+// expose the access token promise
+window.accessTokenPromise = () =>
```

Comment:
Does it need to be on a separate object than `window.Happychat`? If yes, could we prefix it with `happychat` or something indicating it's token promise for HC?